lib/shared/permissions.js

```diff
@@ -145,22 +146,13 @@ exports.elevateCommand = (command, options) => {
   const prefixedCommand = _.concat(exports.getEnvironmentCommandPrefix(options.environment), command);
 
   if (os.platform() === 'win32') {
-    const elevator = Bluebird.promisifyAll(require('elevator'));
-
-    return elevator.executeAsync(prefixedCommand, {
-      hidden: true,
-      terminating: true,
-      doNotPushdCurrentDirectory: true,
-      waitForTermination: true
-    }).then(() => {
+    const elevator = Bluebird.promisifyAll(bindings.load('elevator'));
+    return elevator.elevateAsync(_.concat([
+      'cmd.exe',
```

Comment:
We can refactor the writer proxy to accept most of them, however `ELECTRON_RUN_AS_NODE` will need to be passed as an env var :(